### PR TITLE
v1.14: WG tunneling

### PR DESCRIPTION
**Pull request by brb**
CI passed https://github.com/cilium/cilium/actions/runs/6717864678/job/18256497398 (w/ https://github.com/cilium/cilium-cli/pull/2055).

### CODE REVIEW

**Review by viktor-kurchenko — APPROVED** [review]

**Review by gandro — APPROVED** [review]

File: .github/workflows/conformance-e2e.yaml [hunk 1]
```diff
@@ -348,7 +348,7 @@ jobs:
 
             mkdir -p cilium-junits
 
-      - name: Run tests (${{ join(matrix.*, ', ') }})
+      - name: Run tests
```
Comment (brlbil) [inline .github/workflows/conformance-e2e.yaml:321]: As far as I am concerned step name should match the --junit-property github_job_step name so either step name should be `Run tests (${{ matrix.name }})` or --junit-property should be `github_job_step="Run tests"`

**Review by brlbil — CHANGES_REQUESTED** [review]

File: .github/workflows/conformance-e2e.yaml (hunk 1 above)
Comment (brb) [reply .github/workflows/conformance-e2e.yaml:321]: TIL, thanks! I changed --junit-property to `Run tests`

**Comment by brb** [comment]
/test-backport-1.14

**Review by brlbil — APPROVED** [review]

**Comment by brb** [comment]
I have to drop https://github.com/cilium/cilium/pull/28917/commits/e945a68bdeea616861480dd9faaa6b692389ef9c, as otherwise the ci-e2e is going to be broken until the new CLI version w/ is released https://github.com/cilium/cilium-cli/pull/2055 :sob:

**Comment by brb** [comment]
/test-backport-1.14

**Comment by brb** [comment]
`Conformance External Workloads` is currently broken. Marking ready-to-merge.

**Pull request merged**